prime-router/metadata/fhir_transforms/default-sender-transform.yml

```diff
@@ -0,0 +1,33 @@
+constants:
+  rsext: '"https://reportstream.cdc.gov/fhir/StructureDefinition/"'
```

Comment:
How is this constant used?

Reply:
Is is just used in the software vendor name to put the url part in the resource. So 'Bundle.entry.resource.ofType(MessageHeader).source.extension(%`rsext-software-vendor-org`)'
Will get replaced with 'Bundle.entry.resource.ofType(MessageHeader).source.extension(`https://reportstream.cdc.gov/fhir/StructureDefinition/-software-vendor-org`)'